Automattic/pocket-casts-android
# Update Release Toolkit from `10.0.0` to `11.0.1`

## Description
This PR updates the Release Toolkit from `10.0.0` to `11.01`. `11.0.1` fixes an error when downloading Android translations. 

## Testing Instructions
1. Checkout this PR's branch
2. Copy `pocket-casts-android/fastlane/env/user.env-example` to `~/.pocketcastsandroid-env.default`. None of the values need to be filled in, the file just needs to be present to run the Fastlane lane. 
3. Run `bundle exec fastlane run configure_apply`
4. In the `finalize_release` lane, comment everything out but this: 
```
    android_download_translations(
      res_dir: File.join('modules', 'services', 'localization', 'src', 'main', 'res'),
      glotpress_url: GLOTPRESS_APP_STRINGS_PROJECT_URL,
      locales: SUPPORTED_LOCALES,
      lint_task: 'lintRelease'
    )
```
5. Run `bundle exec fastlane finalize_release`
6. The translations should successfully be downloaded

## Screenshots or Screencast 


## Checklist
 
#### I have tested any UI changes...
